Dockerfile

```diff
@@ -30,6 +30,8 @@ RUN microdnf --nodocs --noplugins --setopt install_weak_deps=0 -y install nginx
 # Log to symlinks in tmp
     && sed -i 's,/var/log/nginx/access.log,/tmp/logs/nginx.access.log,' /etc/nginx/nginx.conf \
     && sed -i 's,/var/log/nginx/error.log,/tmp/logs/nginx.error.log,' /etc/nginx/nginx.conf \
+# Create resolver stub file to be updated in entrypoint
+    && install -o999 -g999 /dev/null /etc/nginx/conf.d/resolvers.conf \
```

Comment:
I would move this down along with the other `install` commands just for _aesthetics_.

https://github.com/RedHatProductSecurity/osim/blob/0a268faff816f90c33d3077a7e14440692b2768e/Dockerfile#L42-L44

Reply:
I put it here because it's grouped with the other /etc/nginx lines, but I don't have a strong opinion on where it goes

Reply:
It's just because it starts with `install -o999 -g999 /dev/null`, doesn't really matter 😆